### PR TITLE
Feat/cli 56

### PR DESCRIPTION
**Pull request by bsmithai**
What this solves:

- issue with container id
- patched config.dump to include new directories

What needs done: 

/run/containers patch, so far we just do a /var/lib patch

### CODE REVIEW

File: api/dump.go [hunk 1]
```diff
@@ -24,6 +24,11 @@ const (
 	sys_pidfd_getfd       = 438
 )
 
+type Bundle struct {
```
Comment (nravic) [inline api/dump.go:27]: Can you add a comment about what a Bundle is/does?

File: api/restore.go [hunk 2]
```diff
@@ -215,17 +206,61 @@ func patchPodmanRestore(opts *container.RuncOpts) error {
 
 func (c *Client) RuncRestore(imgPath, containerId string, opts *container.RuncOpts) error {
 
+	bundle := Bundle{Bundle: opts.Bundle}
+
+	isPodman := checkIfPodman(bundle)
+
+	if isPodman {
```
Comment (nravic) [inline api/restore.go:213]: Also a comment explaining what's happening here w/ the rsync

File: cmd/client-daemon.go [hunk 3]
```diff
@@ -63,7 +63,7 @@ var startDaemonCmd = &cobra.Command{
 
 		logger.Info().Msgf("daemon started at %s", time.Now().Local())
 
-		go startgRPCServer()
+		startgRPCServer()
```
Comment (nravic) [inline cmd/client-daemon.go:66]: why was this removed as a goroutine? IIRC the daemon won't work otherwise because startgRPCServer is blocking

**Review by nravic — COMMENTED** [review]
lgtm, some things I don't understand can be resolved with better comments

File: cmd/client-daemon.go (hunk 3 above)
Comment (Brandonsmith738) [reply cmd/client-daemon.go:66]: I just forgot to re-add this. I drop go daemon whenever I want to debug rpc

Comment (bsmithai) [reply cmd/client-daemon.go]: Forgot to re-add

**Review by nravic — APPROVED** [review]
lgtm
